development/configuration/configuring-prestashop.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Configuring PrestaShop
+---
+
+# Introduction to configuration
+
+## Storage & access
+
+Most of PrestaShop's configuration is stored in the `ps_configuration` database table and is accessible through the [Configuration storage service][configuration-storage].
```

Comment:
Should we use "PREFIX_" instead of "ps_" ? 
```suggestion
Most of PrestaShop's configuration is stored in the `PREFIX_configuration` database table and is accessible through the [Configuration storage service][configuration-storage].
```

Reply:
I saw that elsewhere in the docs and I wonder about what's the best way to document table names. I'm not sure if people are aware that you can change the prefix and that it's not literally `PREFIX`.

Reply:
yes you are right, somebody changing the prefix would know that he has to fetch `hisprefix_` instead of `ps_` 👍